app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsScreen.kt

```diff
@@ -315,7 +323,7 @@ fun ParkingDetailsScreen(
 
               // Images
               Row(
-                  modifier = Modifier.fillMaxWidth().testTag("ImagesRow"),
+                  modifier = Modifier.height(150.dp).fillMaxWidth().testTag("ImagesRow"),
```

Comment:
Static height should be changed to fit the screen

Reply:
Do you mean like what you often do where the you set the dimensions to a fraction of the viewport ?
If that is the case, I prefer keeping 150.dp, less is too small to see the image, and if I adjust the size in relation to the screen height I risk displaying the images not big enough to see anything.

Reply:
Fair point